python/nano/src/bigdl/nano/utils/pytorch/convert.py

```diff
@@ -42,6 +43,17 @@ def export_to_onnx(model, input_sample=None, onnx_path="model.onnx", dynamic_axe
     forward_args = get_forward_args(model)
     tensor_args = get_conditional_args(model)
     input_sample = get_input_example(model, input_sample, forward_args)
+
```

Comment:
add some comment about what line 47-56 is doing, feel free to add an example in the comment. It may take future developer some time to understand :)

Reply:
Sure! I'll leave completed comments later